tests/version.rs

```diff
@@ -20,35 +20,6 @@ fn simple() {
 
 }
 
-#[derive(RustcDecodable)]
-struct FooFlags {
-    flag_version: bool,
-}
-
-fn real_main(flags: FooFlags, _config: &cargo::Config) ->
-        cargo::CliResult<Option<String>> {
-    if flags.flag_version {
-        Ok(Some("foo <version>".to_string()))
-    } else {
-        Ok(None)
-    }
-}
-
-#[test]
-fn subcommand_with_version_using_exec_main_without_stdin() {
-    let usage = "
-Usage: cargo foo [--version]
-
-Options:
-    -V, --version       Print version info
-";
-    let args: Vec<String> = vec!["cargo", "foo", "--version"]
-        .into_iter().map(|s| s.to_string()).collect();
-    let result = cargo::call_main_without_stdin(
-                real_main, &cargo::Config::default().unwrap(),
-                usage, &args, false);
-    assert_eq!(result.unwrap(), Some("foo <version>".to_string()));
-}
```

Comment:
I don't think this is a really useful test now. I guess that it was necessary before `docop` implemented `options_first` flag, but know when it has it we can be sure that docopt will pass through all the arguments after the subcommand.